signed/verifiers.go

```diff
@@ -17,10 +17,9 @@ import (
 // can be injected into a verificationService. For testing and configuration
 // purposes, it will not be used by default.
 var Verifiers = map[string]Verifier{
-	"ed25519": Ed25519Verifier{},
-	"rsa":     RSAVerifier{},
-	"rsassa-pkcs1-v1_5-sign": RSAPemVerifier{}, // RSASSA-PKCS1-V1_5-SIGN
-	"pycrypto-pkcs#1 pss":    RSAPSSVerifier{},
```

Comment:
Leave in this line and the line above for legacy and cross language compatibility reasons.